forge-gui/res/cardsfolder/upcoming/tallyman_of_nurgle.txt

```diff
@@ -0,0 +1,14 @@
+Name:Tallyman of Nurgle
+ManaCost:2 B
+Types:Creature Astartes Warrior
+PT:2/3
+K:Lifelink
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | SVarCompare$ GE1 |  Execute$ TrigBranch | TriggerDescription$ The Seven-fold Chant — At the beginning of your end step, if a creature died this turn, you draw a card and you lose 1 life. If seven or more creatures died this turn, instead you draw seven cards and you lose 7 life.
+SVar:TrigBranch:DB$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GT7 | TrueSubAbility$ DBDrawSeven | FalseSubAbility$ DBDraw
```

Comment:
I think you could do this more cleanly with `X:Count$Compare Y` then you wouldn't need the branch

See `Feed the Clan` and similar